### PR TITLE
[PB-481]: Feat/UI reporting changes

### PR DESCRIPTION
**Pull request by larry-internxt**
- Used webdav events to send changes to sync notifications
- Added renaming logic
- Removed some old sync notification parts

### CODE REVIEW

**Review by PixoDev — CHANGES_REQUESTED** [review]
@larry-internxt no tests no party mate, unit tests work correctly on master-erb, can you give it a look?

File: src/main/analytics/webdav-handlers.ts [hunk 1]
```diff
@@ -10,27 +10,52 @@ function subscribeToFlowEvents(ipc: IpcWebdavFlow) {
...
+    });
+  });
+
   ipc.on('WEBDAV_FILE_DOWNLOADED', (_, payload) => {
-    const { name, type, size, uploadInfo } = payload;
+    const { name, type, size, processInfo } = payload;
+
+    broadcastToWindows('sync-info-update', {
+      action: 'DOWNLOADED',
+      name: name + '.' + type,
```
Comment (PixoDev) [inline src/main/analytics/webdav-handlers.ts:40]: what if file doesn't has type? we display 'filename.' ?

File: src/main/analytics/webdav-handlers.ts [hunk 2]
```diff
@@ -48,39 +73,59 @@ function subscribeToFlowEvents(ipc: IpcWebdavFlow) {
   ipc.on('WEBDAV_FILE_RENAMED', (_, payload) => {
     const { name } = payload;
 
+    broadcastToWindows('sync-info-update', {
+      action: 'RENAMED',
+      name,
```
Comment (PixoDev) [inline src/main/analytics/webdav-handlers.ts:78]: we are not missing the type here?

File: src/main/analytics/webdav-handlers.ts (hunk 1 above)
Comment (larry-internxt) [reply src/main/analytics/webdav-handlers.ts]: nice catch, I've added a new property nameWithExtension for a better management

File: src/main/analytics/webdav-handlers.ts (hunk 2 above)
Comment (larry-internxt) [reply src/main/analytics/webdav-handlers.ts]: on this one, the name comes with extension, so i've renamed it for a better understanding

**Review by PixoDev — APPROVED** [review]

**Pull request merged**